.gitignore

```diff
@@ -3,3 +3,14 @@
 *build*
 
 *.user
+*.code-workspace
+result
+.vscode/c_cpp_properties.json
+CMakeFiles
+include/odri/control/interface/*.hh
+CMakeCache.txt
+CMakeDoxyfile.in
+CMakeDoxygenDefaults.cmake
+DartConfiguration.tcl
+config.h
+config.log
```

Comment:
I guess those ones are expected to go in a `build` dir.
Usually, we use cmake like this:
```
cmake -B build .S .  # configure
cmake --build build  # compile
```